proto/penumbra/penumbra/view/v1alpha1/view.proto

```diff
@@ -185,13 +185,19 @@ message BroadcastTransactionResponse {
 message TransactionPlannerRequest {
   // The expiry height for the requested TransactionPlan
   uint64 expiry_height = 1;
-  // The fee for the requested TransactionPlan, if any.
-  core.component.fee.v1alpha1.Fee fee = 2;
+
+  oneof fee_option {
+    // The fee for the requested TransactionPlan, if any.
+    core.component.fee.v1alpha1.Fee fee = 2;
+    // The fee tier for the requested TransactionPlan, if any.
+    core.component.fee.v1alpha1.FeeTier fee_tier = 3;
+  };
```

Comment:
I think it would be good if this were changed to make it clearer that it's either a manual fee setting or an auto fee setting.